Dockerfile

```diff
@@ -20,6 +20,7 @@ COPY ./dags ${AIRFLOW_HOME}/dags
 
 RUN chown -R airflow: .
 
+
```

Comment:
intentional?

Reply:
yes. adding another line makes it slightly easier to read (in my opinion)